rplugin/python3/deoplete/sources/deoplete_jedi.py

```diff
@@ -60,6 +60,7 @@ def on_init(self, context):
         log_file = ''
         root_log = logging.getLogger('deoplete')
 
+        self.is_debug_enabled = False
```

Comment:
I think the change is not needed.
Please update deoplete to the latest version.